README.md

```diff
@@ -14,3 +16,47 @@ This lib depends on libgraphqlparser (in submodule in this repository).
 
 You need to build and install this libgraphqlparser binary on your system before using this.
 See [here](https://github.com/graphql/libgraphqlparser) for more details.
+
+## Roadmap - Road to v1
+
+Based on our experience to managing GraphQL API in production for more than one year. We decided to work on a brand new GraphQL engine.
```

Comment:
Following on our experience **in** managing GraphQL API**s** in production **for more than a year**, we've decided to build a brand new GraphQL engine.

Reply:
👍